cmd/client/client.go

```diff
@@ -62,6 +63,11 @@ func RegisterClientCommands(cmd *cobra.Command) {
 	cmd.AddCommand(EdgeRuleCmd)
 }
 
+func clientError(err error) {
```

Comment:
rather implement func body as:

```
if err != nil {
        logging.GetLogger().Error(err)
        os.Exit(err)
}
```

Reply:
So the name of the function is clientError so I guess/hope err is not nil